backend/data/migrations/20200321015501_update_users_table.js

```diff
@@ -0,0 +1,21 @@
+module.exports.up = function(knex) {
+  return knex.schema.table("users", users => {
+    users.string("blurb", 255);
+    users.string("country", 255);
+    users.string("city", 255);
+    users.boolean("email_verification").defaultTo(false);
+    users.boolean("phone_verification").defaultTo(false);
+    users.boolean("merchant").defaultTo(false);
```

Comment:
Don't think we need this field

Reply:
I intend to store a short description or summary about each user, limited to 255 characters in length, in the 'blurb' column." In the future, we will have the functionality for users to further narrow their searches down to cities and countries to allow better local trade. I have removed the field that we are not currently using.